site/docs/how-to/review.md

```diff
@@ -5,34 +5,42 @@ slug: '/review'
+The standard practice for code review of database changes is to write the changes via a [migration system](https://en.wikipedia.org/wiki/Schema_migration) such that the changes are executed after approval. We do not recommend that cloud infrastructure changes done with IaSQL follow this pattern for two reasons: first, these migration tools often do not support data changes, only schema changes, and the IaSQL schema is defined by [modules](../concepts/module.md) that can be upgraded and changed over time, and second, you lose all of the feedback the database can give you on whether or not your SQL statements are valid and what impact they will have on your cloud infrastructure.
+
+Instead, we recommend complex, or delicate, infrastructure changes should be treated as transactions in IaSQL. [IaSQL transactions](../concepts/transaction.md) are a bit different from normal DB transactions in that they lock all database changes from propagating into the cloud but do not lock the changes from being viewed by other database connections, making them "multiuser", so others can inspect the changes, and can preview what they would do with the [`iasql_preview`](../modules/builtin/iasql_functions.md) function, similar to Pulumi’s `preview` or Terraform’s `plan`.
+
+But, reviewable artifacts are still very useful to summarize and explain the intent behind a proposed set of changes, so we have created an easy-to-follow yet comprehensive convention to review infrastructure changes within transactions which:
 - works with any cloud identity and access management structure
 - works with IaSQL schema upgrades
 - works with any version control system, not just GitHub, without the need for privileged access
 - preserves the ability to `iasql_preview` changes in development and get the type-safety feedback from the dashboard editor
 
 ```sql title="Create review for an infrastructure change within a transaction"
+-- First, we open a new transaction with `iasql_begin`
 SELECT * FROM iasql_begin();
-
+-- Then we make the infrastructure changes we need.
+-- If a SQL statement is invalid, Postgres will prevent it
+-- from running and it won't end up in the transaction, so
+-- we can use it as our IDE.
 UPDATE log_group SET log_group_name = 'test' WHERE id = 4;
+-- We create the review artifact with `iasql_create_review`, providing
+-- a title and a description of why we're doing this.
+SELECT * FROM iasql_create_review('My infra change', 'Why this change is necessary')
 
-SELECT * FROM iasql_create_review('My review' )
-```
-
-The raw markdown content returned will look as follows:
+The output of `iasql_create_review` is markdown-formatted text and will look as follows:
 
 ```
-# My review
+# My infra change
 
-Review description
+Why this change is necessary
 
 ## IaSQL Preview
 
 | action | table_name | id | description |
 | --- | --- | --- | --- |
-| update | log_group | 4 | quickstart-log-group|us-east-2 |
+| update | log_group | 4 | quickstart-log-group | us-east-2 |
```

Comment:
Btw, why does this have the log group name and the region? There are 5 columns here vs the 4 titled columns.

I think the fix @aguillenv suggested was kinda incorrect: I bet the `log_group` description generation was creating the string `quickstart-log-group|us-east-2` and that is playing havoc with the markdown table generation here.

I think the proper solution is to escape the `|` symbol in the text inserted into the tables and then show the escaped version here, instead.

Reply:
Oh, you are right, I did not realize that in my PR :/

Reply:
This is being handled at https://github.com/iasql/iasql/pull/2349. It produces the following markdown for this case:

# New Review

Review description

## IaSQL Preview

| action | table_name | id | description |
| --- | --- | --- | --- |
| update | log_group | 2 | /aws/codebuild/oow1f-ecr-builder\|us-east-2 |

## SQL changes

```sql

UPDATE log_group
SET log_group_name = '/aws/codebuild/oow1f-ecr-builder', log_group_arn = 'tets', creation_time = '2023-01-25T12:43:59.107+00:00', region = (SELECT region FROM aws_regions WHERE region = 'us-east-2')
WHERE log_group_name = '/aws/codebuild/oow1f-ecr-builder' AND log_group_arn = 'arn:aws:logs:us-east-2:257682470237:log-group:/aws/codebuild/oow1f-ecr-builder:*' AND creation_time = '2023-01-25T12:43:59.107+00:00' AND region = (SELECT region FROM aws_regions WHERE region = 'us-east-2');

```

Reply:
@depombo let's update the docs now that #2349 has landed :)

Reply:
good catch, done :)